advisor-backend/models/event.js

```diff
@@ -30,4 +30,12 @@ export const EventDetail = sequelize.define("EventDetail", {
     type: DataTypes.STRING,
     allowNull: false,
   },
+  time: {
+    type: DataTypes.TIME,
+    allowNull: true,
+  },
+  time_commitment: {
+    type: DataTypes.FLOAT,
```

Comment:
Why does this need to be a FLOAT?

Reply:
In terms of units, time_commitment would be in hours. Allowing for FLOAT, event commitment could be something like 20 minutes or 30 minutes then where time_commitment = 0.33 for approx 20 minutes or 0.5 for 30 minutes, etc

Reply:
I'd recommend storing this as the most granular level of time that you're concerned about (seems like minutes here) or you could use `start_time` and `end_time`. Time fractions often end up being a pain because they don't translate super cleanly (e.g. .3333... vs 20 minutes).